extra-games/polymc/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.2.0
+SRCS="git::commit=tags/$VER::https://github.com/PolyMC/PolyMC"
```

Comment:
```suggestion
SRCS="git::commit=tags/$VER;copy-repo=true::https://github.com/PolyMC/PolyMC"
```

Reply:
LTO issue has been resolved by upgrading PolyMC to 1.2.0.